docs/source/whatsnew/v0.25.rst

```diff
@@ -0,0 +1,24 @@
+Version 0.25 (Date TBD)
+=======================
+
+The new minimum supported versions of dependencies that have been updated are:
+
+* Shapely 2.0
+
+
+Features
+--------
+
+* Ruth Comer introduced `~cartopy.mpl.path.shapely_to_path` and
+  `~cartopy.mpl.path.path_to_shapely` which map a single Shapely geometry or
+  collection to a single Matplotlib path and *vice versa*. (:pull:`2455`)
+
+
+Deprecations and Removals
+-------------------------
+
+* `~cartopy.mpl.patch.path_to_geos` and `~cartopy.mpl.patch.geos_to_path` are
+  deprecated.  Use `~cartopy.mpl.path.path_to_shapely` and
+  `~cartopy.mpl.path.shapely_to_path` instead.
+
+* `cartopy.mpl.patch.path_segments` has moved to `cartopy.mpl.path.path_segments`.
```

Comment:
I'm wondering if we actually want to privatise `path_segments`?  Since it's only two lines it's not hard for downstream to vendor, and it would mean we have more freedom to do what we want with it in future.